File: packages/ios/Sources/CivUI/CivOnThisPageItem.swift
// CivUI — CivOnThisPageItem for SwiftUI
// Single TOC entry. Mirrors civ-on-this-page-item.
//
// Placeholder body — the prop surface satisfies schema parity. See
// .claude/rules/audit-debt.md → "Native platform implementation pass".

import SwiftUI

public struct CivOnThisPageItem: View {
    /// Visible link text.
    public var label: String

    /// Fragment / anchor target.
    public var href: String

    /// Whether this heading is currently in the viewport.
    public var current: Bool

    public init(
        label: String = "",
        href: String = "",
        current: Bool = false
    ) {
        self.label = label
        self.href = href
        self.current = current
    }

    public var body: some View {
        EmptyView()
    }
}

File: packages/android/src/main/kotlin/gov/civui/components/CivOnThisPageItem.kt
// CivUI — CivOnThisPageItem for Jetpack Compose
// Single TOC entry. Mirrors civ-on-this-page-item.
//
// Placeholder body — see .claude/rules/audit-debt.md → "Native platform implementation pass".

package gov.civui.components

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier

@Composable
fun CivOnThisPageItem(
    label: String = "",
    href: String = "",
    current: Boolean = false,
    modifier: Modifier = Modifier,
) {
    Box(modifier = modifier) {
        // Placeholder — see audit-debt.md for native implementation status.
    }
}
